src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -9,7 +10,96 @@ void BgSpot16Bombstone_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808B5A94(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B04(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B6C(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B58(BgSpot16Bombstone* this);
+void func_808B5950(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5934(BgSpot16Bombstone* this);
+void func_808B5AF0(BgSpot16Bombstone* this);
+void func_808B5A78(BgSpot16Bombstone* this);
+
+extern Gfx D_06000C20[];
+extern Gfx D_060009E0[];
+
+typedef struct D_808B5DD8Struct {
```

Comment:
You don't need the struct names for those (only the type ones at the bottom)
```suggestion
typedef struct {
```